rosa_tutorials/rosa-mobb-azure-oidc.adoc

```diff
@@ -0,0 +1,137 @@
+
+image::azure-portal_optional-claims-page.png[Azure Portal - Add Optional Claims Page]
+
+Select ID then check the "email" and "preferred_username" claims and click the "Add" button to configure them for your Azure AD application.
+
+image::azure-portal_add-optional-claims-page.png[Azure Portal - Add Optional Claims - Token Type]
+image::azure-portal_add-optional-email-claims-page.png[Azure Portal - Add Optional Claims - email]
+image::azure-portal_add-optional-preferred_username-claims-page.png[Azure Portal - Add Optional Claims - preferred_username]
+
+When prompted, follow the prompt to enable the necessary Microsoft Graph permissions.
+
+image::azure-portal_add-optional-claims-graph-permissions-prompt.png[Azure Portal - Add Optional Claims - Graph Permissions Prompt]
+
+[id="rosa-mobb-azure-oidc-configure-cluster"]
+== Configure the OpenShift cluster to use Azure AD as the identity provider
+
+Finally, we need to configure OpenShift to use Azure AD as its identity provider. While Red Hat OpenShift Service on AWS (ROSA) offers the ability to configure identity providers via the OpenShift Cluster Manager (OCM), that functionality does not currently support group claims. Instead, we'll configure the cluster's OAuth provider to use Azure AD as its identity provider via the `rosa` CLI. To do so, run the following command, making sure to replace the variable specified:
+
+[source,terminal]
+----
+$ rosa create idp \
+    --cluster ${CLUSTER_NAME} \ <1>
+    --type openid \
+    --name ${IDP_NAME} \ <2> 
+    --client-id ${APP_ID} \ <3>
+    --client-secret ${CLIENT_SECRET} \ <4>
+    --issuer-url https://login.microsoftonline.com/${TENANT_ID}/v2.0 \ <5>
+    --email-claims email \
+    --name-claims name \
+    --username-claims preferred_username \
+    --extra-scopes email,profile
+----
+--
+<1> Provide the name of your ROSA cluster with the `export CLUSTER_NAME=example-cluster` command in your terminal.
+<2> Provide the name you used in the OAuth callback URL with the `export IDP_NAME=AAD` command in your terminal.
+<3> Provide the Application (client) ID with the `export APP_ID=yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy` command in your terminal.
+<4> Provide the client secret with the `export CLIENT_SECRET=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` command in your terminal.
+<5> Provide the Directory (tenant) ID with the `export TENANT_ID=zzzzzzzz-zzzz-zzzz-zzzz-zzzzzzzzzzzz` command in your terminal.
+--
+
+[id="rosa-mobb-azure-oidc-grant-permissions"]
+== Grant additional permissions to individual users
+
+Once the cluster authentication Operator reconciles your changes (generally within a few minutes), you will be able to login to the cluster using Azure AD.
+
+Once you login, you will notice that you have very limited permissions. This is because, by default, OpenShift only grants you the ability to create new projects (namespaces) in the cluster. Other projects (namespaces) are restricted from view.
```

Comment:
```suggestion
Once you log in, you will notice that you have very limited permissions. This is because, by default, OpenShift only grants you the ability to create new projects (namespaces) in the cluster. Other projects (namespaces) are restricted from view.
```